worker/task/process_event_delivery.go

```diff
@@ -269,7 +269,13 @@ func ProcessEventDelivery(endpointRepo datastore.EndpointRepository, eventDelive
 			}
 		}
 
-		err = eventDeliveryRepo.UpdateEventDeliveryWithAttempt(ctx, project.UID, *eventDelivery, attempt)
+		err = attemptsRepo.CreateDeliveryAttempt(ctx, &attempt)
+		if err != nil {
+			log.WithError(err).Error("failed to create delivery attempt", eventDelivery.UID)
+			return &DeliveryError{Err: fmt.Errorf("%s, err: %s", ErrDeliveryAttemptFailed, err.Error())}
+		}
+
+		err = eventDeliveryRepo.UpdateEventDeliveryWithAttempt(ctx, project.UID, *eventDelivery)
```

Comment:
@jirevwe we probably just create a repo method to write directly to the attempts table instead of using this one.

Reply:
We just need to rename it appropriately; we don't update it with the attempt anymore, but we save the latency and status info.

Reply:
I'll up date it to `UpdateEventDeliveryMetadata()`